src/login.rs

```diff
@@ -73,3 +73,28 @@ pub fn get_cgroup(pid: Option<pid_t>) -> Result<String> {
     let cg = unsafe { MString::from_raw(c_cgroup) };
     Ok(cg.unwrap().to_string())
 }
+
+/// Determines the session identifier of a process.
+///
+/// Specific processes can be optionally targeted via their PID. When no PID is
+/// specified, operation is executed for the calling process.
+/// This method can be used to retrieve a session identifier.
+pub fn get_session(pid: Option<pid_t>) -> Result<String> {
+    let mut c_session: *mut c_char = ptr::null_mut();
+    let p: pid_t = pid.unwrap_or(0);
+    sd_try!(ffi::sd_pid_get_session(p, &mut c_session));
+    let cg = unsafe { MString::from_raw(c_session) };
+    Ok(cg.unwrap().to_string())
```

Comment:
@lucab Will do. I'll change the coding style also in other functions in `login.rs`. Thanks!